js/react_native/ios/OnnxruntimeModule.xcodeproj/project.pbxproj

```diff
@@ -46,6 +47,7 @@
 		DBDB57DB2603211A004F16BE /* Info.plist */ = {isa = PBXFileReference; lastKnownFileType = text.plist.xml; path = Info.plist; sourceTree = "<group>"; };
 		DBDB588A2609B18F004F16BE /* Resources */ = {isa = PBXFileReference; lastKnownFileType = folder; name = Resources; path = OnnxruntimeModuleTest/Resources; sourceTree = "<group>"; };
 		DBDB58AF262A92D6004F16BE /* OnnxruntimeModuleTest.mm */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.cpp.objcpp; path = OnnxruntimeModuleTest.mm; sourceTree = "<group>"; };
+		E3D7A60D29F8D87D00FF5265 /* libiconv.tbd */ = {isa = PBXFileReference; lastKnownFileType = "sourcecode.text-based-dylib-definition"; name = libiconv.tbd; path = Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/lib/libiconv.tbd; sourceTree = DEVELOPER_DIR; };
```

Comment:
Where can we find the license of this file: Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/lib/libiconv.tbd ? 

Would it be bad to hardcode a XCode version here?

Reply:
This file was generated from XCode GUI Application. I have to link the library for these 2 xcode projects from the Xcode GUI. The library is part of Apple SDK. 

I can try to remove the 13.3 from the MacOSX13.3.sdk, and it should works.

Reply:
/
<img width="1128" alt="Screenshot 2023-04-25 at 9 25 57 PM" src="https://us
<img width="379" alt="Screenshot 2023-04-25 at 9 26 48 PM" src="https://user-images.githubusercontent.com/73297588/234471153-18dc058c-a0c3-413c-ac45-cb0a6a97fe8e.png">
er-images.githubusercontent.com/73297588/234471134-abf763a6-e864-4078-b245-abe7a7024743.png">

Reply:
<img width="1128" alt="Screenshot 2023-04-25 at 9 25 57 PM" src="https://user-images.githubusercontent.com/73297588/234471194-da956c5b-e27c-43ff-b46c-6beeea5c3060.png">

Reply:
<img width="379" alt="Screenshot 2023-04-25 at 9 26 48 PM" src="https://user-images.githubusercontent.com/73297588/234471214-55d911b1-52ab-49f7-9ae9-028c5d8a5061.png">

Reply:
The library must come with some headers files. One of them should be iconv.h. Could you please open the file and inspect its license?

Reply:
/* Copyright (C) 1999-2003, 2005-2006 Free Software Foundation, Inc.
   This file is part of the GNU LIBICONV Library.

   The GNU LIBICONV Library is free software; you can redistribute it
   and/or modify it under the terms of the GNU Library General Public
   License as published by the Free Software Foundation; either version 2
   of the License, or (at your option) any later version.

   The GNU LIBICONV Library is distributed in the hope that it will be
   useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
   Library General Public License for more details.

   You should have received a copy of the GNU Library General Public
   License along with the GNU LIBICONV Library; see the file COPYING.LIB.
   If not, write to the Free Software Foundation, Inc., 51 Franklin Street,
   Fifth Floor, Boston, MA 02110-1301, USA.  */

Reply:
It is GNU, but  is the copyright expired?

Reply:
No. I don't think so. We should avoid using it.

Reply:
Then we have to find another alternative before we adapt MacOS SDK 13.3.

Reply:
See this: https://stackoverflow.com/questions/57734434/libiconv-or-iconv-undefined-symbol-on-mac-osx/57734435